org.ppsspp.PPSSPP.yml

```diff
@@ -2,6 +2,7 @@ app-id: org.ppsspp.PPSSPP
 runtime: org.freedesktop.Platform
 runtime-version: '22.08'
 sdk: org.freedesktop.Sdk
+separate-locales: false
```

Comment:
> separate-locales (boolean)
 Separate out locale files and translations to an extension runtime. Defaults to true.

Why is this disabled?

Reply:
Because Flatpak’s automatic handling of gettext and Qt messages does not apply to PPSSPP. We should not tempt Flatpak to get *confused* when exporting the bundled *`.Locale` extension runtime. Besides, Flatpak may be extended (and probably will) with some further l10n framework in the future which may then interfere with PPSSPP’s l10n scheme. So, it is better to play safe and thus disable Flatpak’s automatic *`.Locale` extension runtime export feature.